include/depthai/modelzoo/Zoo.hpp

```diff
@@ -1,161 +1,22 @@
 #pragma once
 
-#include <cpr/cpr.h>
-
 #include "depthai/modelzoo/NNModelDescription.hpp"
 
 namespace dai {
-
 constexpr const char* MODEL_ZOO_URL = "https://api.cloud-stg.luxonis.com/graphql";
```

Comment:
```suggestion
constexpr const char* MODEL_ZOO_URL = "https://api.cloud.luxonis.com/graphql";
```

Let's also switch to a production url.